site/rcuinfo.md

```diff
@@ -0,0 +1,46 @@
+## Specifying RCU connection information in the model
+
+During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and requires extra options.
+
+For example, in order to use Oracle Autonomous Transaction Processing Cloud Database for the JRF domain, specify the following information in the model:
```

Comment:
in order to use Oracle Autonomous Transaction Processing Cloud Database -> in order to use the Oracle Autonomous Transaction Processing Cloud Database